src/utils/config/service-helpers.js

```diff
@@ -588,6 +591,16 @@ export function cleanServiceGroups(groups) {
         if (type === "healthchecks") {
           if (uuid !== undefined) cleanedService.widget.uuid = uuid;
         }
+        if (type === "speedtest") {
+          if (bitrateNumOfDecimalPlaces !== undefined) {
+            // if the result can't be parsed, set to -1
+            cleanedService.widget.bitrateNumOfDecimalPlaces = Number.parseInt(bitrateNumOfDecimalPlaces, 10);
+            if (Number.isNaN(cleanedService.widget.bitrateNumOfDecimalPlaces)) {
+              // negative values fallback to the default
+              cleanedService.widget.bitrateNumOfDecimalPlaces = -1;
+            }
+          }
+        }
```

Comment:
```suggestion
        if (type === "speedtest") {
          if (bitratePrecision !== undefined) {
            cleanedService.widget.bitratePrecision = parseInt(bitratePrecision, 10);
          }
        }
```

Dont bother with this if youre going to check the value later anyway.